app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AdminController < ApplicationController
+  def index
+    @shelters = Shelter.find_by_sql("SELECT * FROM shelters ORDER BY name DESC")
```

Comment:
Potentially move this to the shelter model file.  Maybe something we can ask about during our checkin.